Library/Homebrew/cmd/update-bash.sh

```diff
@@ -320,8 +333,23 @@ EOS
   # this procedure will be removed in the future if it seems unnecessary
   rename_taps_dir_if_necessary
 
-  # kill all of subprocess on interrupt
-  trap '{ pkill -P $$; wait; exit 130; }' SIGINT
+  # Spawn ssh-agent if it's not running and at least one of repos is using ssh
+  # https://github.com/Homebrew/homebrew/issues/48267
+  for DIR in "$HOMEBREW_REPOSITORY" "$HOMEBREW_LIBRARY"/Taps/*/*
+  do
+    [[ -d "$DIR/.git" ]] || continue
+    cd "$DIR" || continue
+    [[ "$(git config --get remote.origin.url 2>/dev/null)" =~ ^https?:// ]] && continue
```

Comment:
So there are two formats for [ssh protocol](https://git-scm.com/book/tr/v2/Git-on-the-Server-The-Protocols#The-SSH-Protocol):
* `ssh://user@server/project.git`
* `user@server:project.git`

So I went to check whether `://` exists and `ssh://` does not exist.